deb/ubuntu-xenial/Dockerfile.aarch64

```diff
@@ -7,7 +7,7 @@ RUN apt-get update && apt-get install -y golang-go apparmor bash-completion btrf
 # bootstrap, so we use golang-go (1.6) as bootstrap to build Go from source code.
 # We don't use the official ARMv6 released binaries as a GOROOT_BOOTSTRAP, because
 # not all ARM64 platforms support 32-bit mode. 32-bit mode is optional for ARMv8.
-ENV GO_VERSION 1.8.5
+ENV GO_VERSION 1.9.2
 RUN mkdir /usr/src/go && curl -fsSL https://golang.org/dl/go${GO_VERSION}.src.tar.gz | tar -v -C /usr/src/go -xz --strip-components=1 \
```

Comment:
Go 1.8.5 and above is iavailable as binary tarball, so it's no longer needed to compile the aarch64 variant from source.

Reply:
Ah, yes; see my PR upstream for the Go 1.8.5 bump; https://github.com/moby/moby/pull/35307/files